lib/Mail/Milter/Authentication/Handler/DKIM.pm

```diff
@@ -252,19 +252,20 @@ sub eom_callback {
               )
             {
 
-                my $key_size = 0;
-                my $key_type = q{};
                 my $selector = eval{ $signature->selector } // q{};
-                eval {
-                    my $key = $signature->get_public_key();
-                    $key_size = $key->size();
-                    $key_type = $key->type();
-                };
+                my $key = eval {$signature->get_public_key()};
+                my $key_type = eval{ $key->type() } // 'unknown';
+                # Key size for ed21159 does not make sense
```

Comment:
I've not heard of ed21159, so double checking it's a thing versus a typo and meaning ed25519

Reply:
good catch